r/src/main/scala/scala/Console.scala

```diff
@@ -0,0 +1,486 @@
+/*                     __                                               *\
```

Comment:
@jeffsteinmetz thank you very much for pointing this out!

We need to make sure it get listed in the NOTICE, as well as all other files identified as "files forked from other open-sourced projects" which, I assume are all the files excluded from automated licence check in [rat-plugin](https://github.com/apache/incubator-zeppelin/pull/208/files#diff-16eca63f53c817ad231bb79821f6a33dR182)